recap-derive/src/lib.rs

```diff
@@ -66,23 +69,17 @@ pub fn derive_recap(item: TokenStream) -> TokenStream {
             /// Recap derived method. Returns true when some input text
             /// matches the regex associated with this type
             pub fn is_match(input: &str) -> bool {
-                recap::lazy_static! {
-                    static ref RE: recap::Regex = recap::Regex::new(#regex)
-                        .expect("Failed to compile regex");
-                }
                 RE.is_match(input)
             }
         }
     };
 
-    let injector = Ident::new(
-        &format!("RECAP_IMPL_FOR_{}", item.ident.to_string()),
-        Span::call_site(),
-    );
+    let injector = Ident::new(&format!("RECAP_IMPL_FOR_{}", item.ident), Span::call_site());
```

Comment:
This is just removing the `.to_string()` since clippy called it out as unnecessary.